man/man5/zpool-features.5

```diff
@@ -925,5 +925,27 @@ is used to checkpoint the pool.
 The feature will only return back to being \fBenabled\fR when the pool
 is rewound or the checkpoint has been discarded.
 
+
+.sp
+.ne 2
+.na
+\fBhead_errlog\fR
+.ad
+.RS 4n
+.TS
+l l .
+GUID	com.delphix:head_errlog
+READ\-ONLY COMPATIBLE	no
+DEPENDENCIES	none
+.TE
+
+This feature enables the upgraded version of errlog. This required on disk
+error format change. Now, errorlog of each head dataset is stored
+seprately in zap object and primarily keyed by this head Id.
+By this feature every dataset affected by an error block is listed under
```

Comment:
I think this sounds better as:
"With this feature enabled, every dataset affected by an error block is listed in the output of
\fBzpool status\fR.